includes/templates/responsive_classic/css/stylesheet.css

```diff
@@ -57,19 +54,19 @@ HR {height: 1px;margin-top:0.5em;border:none;}
 CODE {font-family:arial, verdana, helvetica, sans-serif;font-size:1em;}
 FORM, SELECT, INPUT {display:inline;font-size:1em;margin:0 0.1em 0 0;}
 input{height:35px;padding-left:5px;margin-bottom:10px;}
-select{border:none;padding-left:5px;width:51%;}
+select{padding-left:5px;width:51%;}
 TEXTAREA {margin:auto;display:block;width:66%;}
 input:focus, select:focus, textarea:focus {}
 legend{padding:0;display:table;text-align:center;width:100%;padding:10px 0;font-size:1.3em;}
-fieldset{border:0;padding:0.01em 0 0 0;padding-bottom:20px;padding-top:20px;margin:0;min-width:0;margin-bottom:20px;}
+fieldset{padding:20px 0;margin:0 0 20px 0;min-width:0;}
 body:not(:-moz-handler-blocked) fieldset{display:table-cell;}
 fieldset > fieldset{}
-fieldset fieldset legend{text-align:left;padding:0;margin:0;position:relative;left:20px;font-size:120%;display:table;background:none;}
+fieldset fieldset legend{text-align:left;padding:0;margin:0;position:relative;left:20px;font-size:120%;display:table;}
 fieldset p{margin-left:20px;}
 fieldset label, fieldset .inputLabel, fieldset h3, fieldset address, #advSearchDefault input, fieldset .buttonRow.back, fieldset select, fieldset input[type=radio], fieldset input[type=checkbox], fieldset div{margin-left:20px;}
 fieldset div, fieldset .buttonRow.forward{margin-right:20px;}
 option{font-size:110%;}
-LABEL, h4.optionName {line-height:1.5em;padding:0.2em;}
+LABEL {line-height:1.5em;padding:0.2em;}
```

Comment:
I'm not seeing where `h4.optionName` (or `h4` or `.optionName`) is ever assigned `line-height` property elsewhere in any of the RC CSS.

Reply:
Argh. Yes, found it.